docs/content/installation/upgrade.md

```diff
@@ -13,5 +13,6 @@ Use this information to upgrade the IBM® block storage CSI driver.
 
 - To manually upgrade the CSI (Container Storage Interface) driver from a previous version with GitHub, perform step 1 of the [installation procedure](install_driver_github.md) for the latest version.
 
-
-
+**Important:** To enable the optional host definer feature when upgrading from IBM® block storage CSI driver 1.9.0 or earlier, the host definer must be manually installed. Once the host definer is initially installed, the host definer automatically updates along with the driver version update during seamless upgrade. 
+  - For manual installation of the host definer with GitHub and OperatorHub.io, see [Installing the host definer](install_hostdefiner.md)
```

Comment:
according to this articulation, a user that installs the driver with the web console (with seamless upgrade set), and installs the host definer with GitHub, might expect the host definer to be seamlessly upgraded (to e.g. 1.11.0).
how about:
```suggestion
**Important:** To enable the optional host definer feature when upgrading from IBM® block storage CSI driver 1.9.0 or earlier, the host definer must be manually installed. Once the host definer is initially installed with the web console, the host definer automatically updates along with the driver during seamless upgrade. 
  - For manual installation of the host definer with GitHub and OperatorHub.io, see [Installing the host definer](install_hostdefiner.md)
```

Reply:
also, for the first sentence, if the focus is on the installation being manual:
```suggestion
**Important:** To enable the optional host definer feature when upgrading from IBM® block storage CSI driver 1.9.0 or earlier, the host definer must be installed manually. Once the host definer is initially installed, the host definer automatically updates along with the driver version update during seamless upgrade. 
  - For manual installation of the host definer with GitHub and OperatorHub.io, see [Installing the host definer](install_hostdefiner.md)
```

Reply:
@matancarmeli7 is the seamless upgrade together with host definer also valid with opertorhub.io?

Reply:
@oriyarde regarding the first section "manually installed" this is fine how it is

Reply:
In this case, I suggest --
I think that this covers the issue better.
@oriyarde  @matancarmeli7  please approve

**Important:** To enable the optional host definer feature when upgrading from IBM® block storage CSI driver 1.9.0 or earlier, _or_ when conducting a manual upgrade, the host definer must be manually installed. If the CSI driver has automatic updates enabled, once the host definer is installed initially, it will automatically update along with the driver version update during future seamless upgrades.

Reply:
That is good to note in the installation section then as well. I will add a note.

This is my new suggestion. (changes from original in bold)

To enable the optional host definer feature when upgrading from IBM® block storage CSI driver 1.9.0 or earlier**, or if the host definer was installed with GitHub,** the host definer must be manually installed. Once the host definer is initially installed, the host definer automatically updates along with the driver version update during seamless upgrade, **if seamless upgrade is set**.

Reply:
>**or if the host definer was installed with GitHub**, the host definer must be manually installed.

if the host definer was installed, it doesn't need to be installed again. I would not include this change (as I wrote, the issue is just with the last sentence).


>Once the host definer is initially installed, the host definer automatically updates along with the driver version update during seamless upgrade, **if seamless upgrade is set**.


this is still not true. when installing a component (driver/host definer/etc...) with GitHub, said component will not be upgraded automatically, even if seamless upgrade is set (for the components that were installed without GitHub).

please use this:
![image](https://user-images.githubusercontent.com/57443811/178513381-e12316b8-b804-4f49-a7a5-76d397d11fc8.png)
to make the changes appear in highlight

Reply:
I made updates to this file and added a note to the install_hostdefiner.md